bc_obps/registration/migrations/0012_facility_alter_operation_table_comment_and_more.py

```diff
@@ -0,0 +1,307 @@
+                ('archived_at', models.DateTimeField(blank=True, null=True)),
+                ('name', models.CharField(db_comment="An operation or facility's name", max_length=1000)),
+                ('type', models.CharField(db_comment="An operation or facility's type", max_length=1000)),
+                (
+                    'id',
+                    models.UUIDField(
+                        db_comment='Primary key to identify the facility',
+                        default=uuid.uuid4,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name='ID',
+                    ),
+                ),
+                ('new_entrant', models.BooleanField(db_comment='Whether or not the facility is a new entrant')),
+                (
+                    'address',
+                    models.ForeignKey(
+                        blank=True,
+                        db_comment='The address of the operation or facility',
+                        null=True,
+                        on_delete=django.db.models.deletion.DO_NOTHING,
+                        related_name='%(class)s_address',
+                        to='registration.address',
+                    ),
+                ),
+                (
+                    'archived_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_archived',
+                        to='registration.user',
+                    ),
+                ),
+                (
+                    'created_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_created',
+                        to='registration.user',
+                    ),
+                ),
+                (
+                    'updated_by',
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name='%(class)s_updated',
+                        to='registration.user',
+                    ),
+                ),
+            ],
+            options={
+                'db_table': 'erc"."facility',
+                'db_table_comment': 'Facilities ',
+            },
+        ),
+        migrations.AlterModelTableComment(
+            name='operation',
+            table_comment='Operations',
+        ),
+        migrations.AlterField(
+            model_name='operation',
+            name='address',
+            field=models.ForeignKey(
+                blank=True,
+                db_comment='The address of the operation or facility',
```

Comment:
ditto here, if this comment is on a field on the operation table should it just say 'operation'?
```suggestion
                db_comment='The address of the operation',
```